src/features/auth/Login.tsx

```diff
@@ -61,7 +61,8 @@ export default function Login({
   const presentToast = useAppToast();
   const dispatch = useAppDispatch();
   const [servers] = useState(getCustomServers());
-  const [server, setServer] = useState(servers[0]);
```

Comment:
This doesn't work quite right. When logging in with sh.itjust.works for example, the instance is technically selected (you can press next), but there's no indication its selected in the list because its not visible.

Reply:
It was working for me, when the restriction of 3 instances is removed from the list:

![image](https://github.com/aeharding/voyager/assets/715417/692bdd67-9730-4e5a-beab-0f7fc26891cb)